Sources/Atomic.swift

```diff
@@ -131,7 +131,7 @@ internal struct UnsafeAtomicState<State: RawRepresentable>: AtomicStateProtocol
 /// The wrapper is a struct so that it can be inlined when it is contained as
 /// part of a reference type, and dodges an extra level of calling indirection
 /// that would otherwise happen if the wrapper itself is an reference type.
-internal struct PosixThreadMutex {
+internal struct PthreadMutex {
```

Comment:
Well, it is generally called pthread anyway... So let's make it a bit shorter...